src/wallet/wallet.cpp

```diff
@@ -426,6 +427,45 @@ std::set<uint256> CWallet::GetConflicts(const uint256& txid) const
     return result;
 }
 
+void CWallet::Flush(bool shutdown)
+{
+    bitdb.Flush(shutdown);
+}
+
+bool CWallet::Verify(const std::string& walletFile, std::string& warningString, std::string& errorString)
+{
+    if (GetBoolArg("-disablewallet", false))
+        return true;
+
+    uiInterface.InitMessage(_("Verifying wallet..."));
+
+    std::string strError;
+    if (!CWalletDB::VerifyEnvironment(walletFile, GetDataDir().string(), strError))
+        errorString += strError;
+        return true;
```

Comment:
missing `if` braces `{}`. Everything after this `return true;` is unreachable code.
Plus, it should return false.

Reply:
Good catch. I recall how and when I messed this up. It's an overlook from backporting https://github.com/bitcoin/bitcoin/pull/8574 which uses the oneliner `return InitError(strError);`